src/SignalR/clients/ts/signalr/src/DefaultHttpClient.ts

```diff
@@ -4,18 +4,21 @@
 import { AbortError } from "./Errors";
 import { FetchHttpClient } from "./FetchHttpClient";
 import { HttpClient, HttpRequest, HttpResponse } from "./HttpClient";
+import { MessageHeaders } from "./IHubProtocol";
 import { ILogger } from "./ILogger";
 import { Platform } from "./Utils";
 import { XhrHttpClient } from "./XhrHttpClient";
 
 /** Default implementation of {@link @microsoft/signalr.HttpClient}. */
 export class DefaultHttpClient extends HttpClient {
     private readonly httpClient: HttpClient;
+    private readonly headers: MessageHeaders;
 
     /** Creates a new instance of the {@link @microsoft/signalr.DefaultHttpClient}, using the provided {@link @microsoft/signalr.ILogger} to log messages. */
-    public constructor(logger: ILogger) {
+    public constructor(logger: ILogger, headers: MessageHeaders) {
```

Comment:
Since default headers are always passed into the call to send, why do we need to store them in DefaultHttpClient?

Reply:
@halter73 Fixed: 206a8db3229af457c8526f0f05d77bc6498f842a